.pipelines/.vsts-vhd-windows-automation.yaml

```diff
@@ -16,13 +16,16 @@ parameters:
   type: boolean
   default: false
 
+variables:
+  - group: "AKS Dev Assistant (KV)"
+
 steps:
 - bash: |
         az extension add -n azure-devops
         echo $MAPPED_ADO_PAT | az devops login --organization=https://dev.azure.com/msazure
         az devops configure --defaults organization=https://dev.azure.com/msazure project=CloudNativeCompute
   env:
-        MAPPED_ADO_PAT: $(ADO_PAT)
```

Comment:
where is this one defined ? should it be removed from the pipeline var ? or from the other KV group ?

Reply:
yes, we will remove it from the pipeline vars once the changes are made